packages/dashboards/src/lib/configurator/services/configurator-data-source-manager.service.ts

```diff
@@ -19,25 +19,15 @@ export class ConfiguratorDataSourceManagerService implements OnDestroy {
     public busy$: BehaviorSubject<boolean> = new BehaviorSubject<boolean>(false);
     public dataSourceFields: BehaviorSubject<(Array<IDataField>)> = new BehaviorSubject<Array<IDataField>>([]);
```

Comment:
I don't know if it would be considered a breaking change or not, but it would be nice if all of the subject members of this class were suffixed with `$` to be consistent (`dataSourceCreated$`, `error$`, `dataSourceFields$`).